### PR TITLE
Validations

### PR DESCRIPTION
**Pull request by VitorGuedesMadeira**
### Blog App: validations and model specs
[Project Requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md)

**_Learning objectives:_**
✔️ Use validations for models.
✔️ Write unit tests for models.

**_author:_** [Vitor Guedes Madeira](https://github.com/VitorGuedesMadeira)

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -4,13 +4,18 @@ class Post < ApplicationRecord
   has_many :likes
   belongs_to :user
 
+  validates :title, presence: true, length: { in: 2..50 }
```
Comment (omar-muhamad) [inline app/models/post.rb]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Title%20must%20not%20exceed%20250%20characters.), Please ensure that Title must not exceed 250 characters, not 50 characters like you did.

File: spec/models/post_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  before(:example) do
+    @new_post = Post.create(user: subject, title: 'Hello', text: 'This is my first post', comments_counter: 0,
+                            likes_counter: 0)
+  end
+
+  context 'Validations should be working' do
+    it 'ALL validations should return true' do
+      expect(@new_post).to be_valid
+    end
```
Comment (omar-muhamad) [inline spec/models/post_spec.rb:17]: - [ ] Please add a test case for every validation in the post model:
- Title must not be blank.
- Title must not exceed 250 characters.
- CommentsCounter must be an integer greater than or equal to zero.
- LikesCounter must be an integer greater than or equal to zero.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @VitorGuedesMadeira,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @VitorGuedesMadeira 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done on making the required changes from the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**